data/json/items/ammo/5x50.json

```
@@ -38,7 +38,7 @@
     "id": "reloaded_5x50dart",
     "copy-from": "5x50dart",
     "type": "AMMO",
-    "name": { "str": "5x50mm flechette, reloaded" },
+    "name": { "str": "5x50mm flechette, reloaded", "str_pl": "5x50mm flechettes, reloaded" },
```

```suggestion
    "name": { "str_sp": "5x50mm flechette, reloaded" },
```
For consistency with other flechette rounds.

---

In the other flechette rounds, they actually have multiple flechettes in each round. This one only has one per round.